admin_manual/configuration/files/encryption_configuration.rst

```diff
@@ -68,16 +68,25 @@ The ownCloud admin and the storage admin will see only encrypted files when brow
    exploitable by anyone with administrator access to your server. For more
    information, read: `How ownCloud uses encryption to protect your data
    <https://owncloud.org/blog/how-owncloud-uses-encryption-to-protect-your-data/>`_.
-   
+  
+Encryption Types
+----------------
+
+ownCloud provides two encryption types:
+
+- **Basic encryption:** every user has their own private/public key pairs, and the private key is protected by the user's password.
+
+- **Master Key:** there is only on key (or key pair) and all files are encrypted using key pair.
```

Comment:
"...on key..." > "...one key..."

"...using key pair." > "using that key pair" / "using the same keys."